### ISSUE TITLE
Book 05: Sentence 050

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#45]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/cbc1a867-9297-4a7d-85be-b2829af15d4c)

```
# sent_id = Book_05_-_The_Omens.txt:50
# text = Injc iuh conaoa in, qujlmach iuhqujn ic onepatilo, ic queleltia, ic qujlochtia, ic qujiacatzacuilia, in jtenotzaliz: injc amo inpan neltiz, ic qujntzatzilia.
# text[norm] = Inic iuh conahua in, quilmach iuhquin ic onepatilo, ic queleltia, ic quilochtia, ic quiyacatzacuilia, in itenotzaliz: inic amo inpan neltiz, ic quintzatzilia.
# text[orig] = Injc·iuh¶conaoa·in·,·qujlmachiuhqujn·icone¶patilo·,·icqueleltia·,·ic·qujlochtia·,·ic¶qujiacatzacuilia·,·inj·tenotzaliz·:·injc¶amo·inpan·neltiz·,·icqujn·tzatzilia·.
# text[osp] = Dezian; que por esto le iniuriauan desta manera: para escaparse del mal aguero, que pronosticaua: y para no ser obligados a cumplir, su llamamjento.
# text[osp_norm] = Decían; que por esto le injuriaban de esta manera: para escaparse del mal agüero, que pronosticaba: y para no ser obligados a cumplir, su llamamiento.
# references = A&D/169:6
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 30.00%
1	Injc	inic	SCONJ	_	_	3	_	_	_
2	iuh	iuh	ADV	_	_	3	_	_	_
3	conaoa	ahua	VERB	_	Movement=And|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	9	_	_	_
4	in	in	DET|SCONJ	_	_	9	_	_	_
5	,	,	PUNCT	_	_	9	punct	_	_
6	qujlmach	quilmach	ADV	_	_	9	_	_	_
7	iuhqujn	iuhquin	ADV	_	_	9	_	_	_
8	ic	ic	ADV	_	_	9	_	_	_
9	onepatilo	pati	VERB	_	_	0	root	_	_
10	,	,	PUNCT	_	_	12	punct	_	_
11	ic	ic	ADV	_	_	12	_	_	_
12	queleltia	elleltia	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	9	_	_	_
13	,	,	PUNCT	_	_	15	punct	_	_
14	ic	ic	ADV	_	_	15	_	_	_
15	qujlochtia	ilochtia	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	12	_	_	_
16	,	,	PUNCT	_	_	18	punct	_	_
17	ic	ic	ADV	_	_	18	_	_	_
18	qujiacatzacuilia	yacatzacuilia	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	12	_	_	_
19	,	,	PUNCT	_	_	21	punct	_	_
20	in	in	DET|SCONJ	_	_	21	_	_	_
21	jtenotzaliz	tenotzaliztli	NOUN	_	Number[psor]=Sing|Person[psor]=3|State=Pos	18	_	_	_
22	:	:	PUNCT	_	_	26	punct	_	_
23	injc	inic	SCONJ	_	_	26	_	_	_
24	amo	amo	ADV	_	_	26	_	_	_
25	inpan	pan	NOUN	_	NounType=Relat|Number[psor]=Plur|Person[psor]=3|State=Pos	26	_	_	_
26	neltiz	nelti	VERB	_	_	12	_	_	_
27	,	,	PUNCT	_	_	29	punct	_	_
28	ic	ic	ADV	_	_	29	_	_	_
29	qujntzatzilia	tzatzilia	VERB	_	_	26	_	_	_
30	.	.	PUNCT	_	_	9	punct	_	_
```

**Comment by ftyers** [comment]
López Austin:
> Así lo reprendían para que en esta forma se ahuyentara enseguida el augurio; así lo impedían, así devolvían, así atajaban su llamado; por esto no se realizaba en ellos lo que les cantaba.

Notes:
- *iuhquin*: #13

**Comment by ftyers** [comment]
Update:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/3d475531-a93e-46f2-b30f-9113ba036eb3)

Questions:
- What is *ic*? It appears a lot...
- How do we deal with *iuhquin* ?
- What is the root? Basically everything has an *in* before it (if we consider *iuhquin* to be *iuhqui* + *in*)

**Comment by Lguyogiro** [comment]
> What is ic? It appears a lot...

Yeah....Andrews's discussion is on pages 450-453. He treats it as a reln that can only take 3rd-person singular possessor.
![imagen](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/b9301db5-39a1-4951-ba84-1fa768886433)
Given the wide range of metaphorical meanings it is a challenge to analyze it. Elsewhere we have treated it as an `ADV` I think (e.g. ordinal construction) and I don't hate that.  I don't see _ic_ taking any nominal morphology other than its _i-_. Also used with nouns of size/length/shape in measurement constructions ( _in intlaxcal cemmolicpitl catca in ic yahualtic_ "their tortillas were an ell round")

So in this sent, if we say it is a reln, I think the nominal/clausal complements are omitted.

**Comment by Lguyogiro** [comment]
> How do we deal with iuhquin ?

I think we said it should be `ADJ` (in #13 )? Also split into _iuhqui n_. I think the syntactic analysis is good in the most recent tree.

This is strange, but I have a feeling that _in,_ before _iuhquin_ is actually a demonstrative pronoun _īn_. So that part is "Así reprendían a ese"  or something like that. Unfortunately we don't have the vowel length to help distinguish. The only sort-of evidence is that, if _in_ is in fact a subordinator there, then we don't have a root. haha

**Comment by ftyers** [comment]
> So in this sent, if we say it is a reln, I think the nominal/clausal complements are omitted. The tree looks correct above with respect to in, but we should decide whether we like ADV or NOUN+NounType=Rel

It could also be that some *in*s are omitted, because in a lot of cases it looks like a subordinator too... 

> This is strange, but I have a feeling that _in,_ before _iuhquin_ is actually a demonstrative pronoun _īn_. So that part is "Así reprendían a ese" or something like that. Unfortunately we don't have the vowel length to help distinguish. The only sort-of evidence is that, if _in_ is in fact a subordinator there, then we don't have a root. haha

Yeah, that is a good point I was thinking that too,

**Comment by ftyers** [comment]
So like this? 
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/5af42adb-12de-47ba-86a9-fa8c1542280a)

**Comment by Lguyogiro** [comment]
LGTM

**Comment by ftyers** [comment]
Merged in f762d1b.
